dask/dataframe/io/tests/test_parquet.py

```diff
@@ -3580,3 +3587,32 @@ def test_custom_metadata(tmpdir, engine):
             custom_metadata=custom_metadata,
         )
     assert "User-defined key/value" in str(e.value)
+
+
+@pytest.mark.parametrize("gather_statistics", [True, False])
+def test_ignore_metadata_file(tmpdir, engine, gather_statistics):
+    tmpdir = str(tmpdir)
+    df = pd.DataFrame({"a": range(100), "b": ["dog", "cat"] * 50})
+    ddf1 = dd.from_pandas(df, npartitions=2)
+    ddf1.to_parquet(path=tmpdir, engine=engine)
+    if engine != "pyarrow-legacy":
+        ddf2 = dd.read_parquet(
+            tmpdir,
+            engine=engine,
+            ignore_metadata_file=True,
+            gather_statistics=gather_statistics,
+        )
+
+        if gather_statistics is False:
+            # Should not have known divisions if gather_statistics=None
+            # Otherwise, we must have used a _metadata file
+            assert set(ddf2.divisions) == {None}
```

Comment:
This seems fine, but I wonder if there's a more direct check we can have that `_metadata` wasn't used. For example, we could manually delete the `_metadata` file before calling `read_parquet`

Reply:
Without the file, we would presumably get the same behaviour whether you have ignore-metadata true or false.

Reply:
I updated the test to do something pretty simple: We write two datasets (one with _metadata, and one without), and check that the result is the same with `ignore_metadata_file`.  How does that sound?

Reply:
I'll defer to you two on this test. I think I still need to wrap my head around how we handle gathering metadata and how that translates to division information on the resulting DataFrame. If you two are happy with this, then let's go with it

Reply:
Another option might be to create a dataset with a "wrong" `_metadata` file (eg only listing a subset of the files, or simply an invalid parquet file), and then reading it with `ignore_metadata_file`, you can be sure it was not used if you get the correct result.

Reply:
That's the answer, thanks @jorisvandenbossche - replace the _metadata file with something invalid (perhaps empty?)

Reply:
That seems like a good approach, I pushed de44f6a (where I copied in a "wrong" _metadata file) and then b472c2b (where I wrote out an "invalid" file).  I'm going with the "invalid" approach for now, but happy to tweak things again.